loopdb/store.go

```diff
@@ -89,33 +89,39 @@ func NewBoltSwapStore(dbPath string, chainParams *chaincfg.Params) (
 		}
 	}
 
+	path := filepath.Join(dbPath, dbFileName)
+	dbExists := fileExists(path)
```

Comment:
Hm, ok. In `lnd` we do the same thing. If I exit `lnd` right after opening the db and then start `lnd` normally, it sees the (empty) db there and starts to apply all migrations. But is there nothing to migrate because we started with no db.

Reply:
Changed to use presence of meta bucket as an indication that the db is already initialized.